packages/components/src/components/inline-message/inline-message.tsx

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import classNames from 'classnames';
```

Comment:
@farhan-nurzi-deriv Please revet this file changes, We shouldn't let devs pass classNames to the design component and override and change the style, We will lose the inconsistency.

If the design is different we should ask designers to fix it on their side, Other than that the parent of this component should be responsible for the styling.

reference: https://www.figma.com/file/pRYAof3hNnDcMbKx4O30Cv/Trading-system?node-id=7125%3A262987&mode=dev

Reply:
Alright @farzin-deriv , i will try the other way. Thanks for letting me know